src/iOS.Autofill/LoginSearchViewController.cs

```diff
@@ -36,6 +36,7 @@ public async override void ViewDidLoad()
             if (!ThemeHelpers.LightTheme)
             {
                 SearchBar.KeyboardAppearance = UIKeyboardAppearance.Dark;
+                SearchBar.ApplyDarkThemesColors();
```

Comment:
I'd change the name to `ApplyDarkTheme()`, given that then we may need to update other things and there should be all in the same method. Now it's the same but IMO theming should be always called in a general way and in the extension or helper class call any particular things for the theme; so the caller doesn't need to call multiple methods to apply the whole theme (e.g. ApplyDarkThemeColors, ApplyDarkThemeIcons, ApplyDarkThemeSeparators, etc)